tests/test_ble_interface_core.py

```
@@ -6146,6 +6425,109 @@ def start_notify(self, *_args: object, **_kwargs: object) -> None:
+            _safe_execute=MagicMock(),
+        )
+        monkeypatch.setattr(
+            iface,
+            "_log_radio_handler",
+            lambda _sender, _data: (_ for _ in ()).throw(RuntimeError("handler boom")),
+            raising=True,
+        )
+        monkeypatch.setattr(
+            iface,
+            "_report_notification_handler_error",
+            lambda msg: errors.append(msg),
```

Ruff (0.15.6)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)